AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Prefix.txt

```diff
@@ -0,0 +1,14 @@
+// swift-tools-version:5.9
```

Comment:
Generated code now goes above the static content, so this separate "prefix" file is needed to provide the copyright notice, imports, etc.